SandboxUwp/CustomDataType.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Collections.Generic;
+using System.Xml;
+using Microsoft.HealthVault.ItemTypes;
+
+namespace SandboxUwp
+{
+    class CustomDataType : ApplicationSpecific
+    {
+        // Add custom properties here
+        public long StartTime { get; set; }
+        public long EndTime { get; set; }
+        public List<double> Values { get; set; }
+        // Custom properties end
+
+        private HealthServiceDateTime m_when;
+
+        // HealthVault expects a specific data schema - custom properties are added at the end
+        public override void WriteXml(XmlWriter writer)
```

Comment:
>override [](start = 15, length = 8)

We should also override ParseApplicationSpecificXml rather than placing the parsing logic in the UI codebehind.